start_cerberus.py

```diff
@@ -31,6 +31,7 @@ def main(cfg):
     if os.path.isfile(cfg):
         with open(cfg, 'r') as f:
             config = yaml.full_load(f)
+        distribution = config["cerberus"].get("distribution", "openshift")
```

Comment:
Maybe we could do `distribution = config["cerberus"].get("distribution", "openshift").lower()` to avoid lower() operation at other places.

Reply:
Fixed it, thanks.